<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Параметры Механизма</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Длина L1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_L1">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
           <property name="value">
            <double>60.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Длина L2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_L2">
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Длина L3</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_L3">
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Угол phi </string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_phi">
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Частота</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox">
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Механизм</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="DeviceView" name="deviceView" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>График</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="PlotView" name="widget_2" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Анимация</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_9"/>
    <addaction name="action_11"/>
    <addaction name="action_10"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Вид</string>
    </property>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Масштаб</string>
     </property>
     <addaction name="action_20"/>
     <addaction name="action_21"/>
     <addaction name="action_22"/>
    </widget>
    <widget class="QMenu" name="menu_7">
     <property name="title">
      <string>Панель инструментов</string>
     </property>
     <addaction name="action_23"/>
     <addaction name="action_24"/>
     <addaction name="action_25"/>
    </widget>
    <addaction name="menu_7"/>
    <addaction name="menu_6"/>
    <addaction name="action_16"/>
    <addaction name="action_17"/>
    <addaction name="action_18"/>
    <addaction name="action_19"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStep_back"/>
   <addaction name="actionStart"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Сохранить механизм</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Загрузить параметры</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Настройки программы</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Первоначальные настройки</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/book-question.png</normaloff>:/icons/icons/book-question.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/control.png</normaloff>
     <normalon>:/icons/icons/control-pause.png</normalon>:/icons/icons/control.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/control-pause.png</normaloff>:/icons/icons/control-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStep_back">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/control-double-180.png</normaloff>:/icons/icons/control-double-180.png</iconset>
   </property>
   <property name="text">
    <string>step back</string>
   </property>
   <property name="toolTip">
    <string>Step back</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Предыдущи кадр</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Старт</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Крутить механизм в другую сторону</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Следующий кадр</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>Сброс</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Стоп</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>Показать сетку</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>Показать обозначения</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>Показать траекторию</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>Показать параметры точек</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>Увеличить</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>Уменьшить</string>
   </property>
  </action>
  <action name="action_22">
   <property name="text">
    <string>Исходный формат</string>
   </property>
  </action>
  <action name="action_23">
   <property name="text">
    <string>Маштаб</string>
   </property>
  </action>
  <action name="action_24">
   <property name="text">
    <string>Анимация</string>
   </property>
  </action>
  <action name="action_25">
   <property name="text">
    <string>Отображения</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DeviceView</class>
   <extends>QWidget</extends>
   <header>deviceview.h</header>
   <container>1</container>
   <slots>
    <slot>setL1(double)</slot>
    <slot>setL2(double)</slot>
    <slot>setL3(double)</slot>
    <slot>setPhi(double)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>PlotView</class>
   <extends>QWidget</extends>
   <header>plotview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>doubleSpinBox_L1</sender>
   <signal>valueChanged(double)</signal>
   <receiver>deviceView</receiver>
   <slot>setL1(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_L2</sender>
   <signal>valueChanged(double)</signal>
   <receiver>deviceView</receiver>
   <slot>setL2(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_L3</sender>
   <signal>valueChanged(double)</signal>
   <receiver>deviceView</receiver>
   <slot>setL3(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_phi</sender>
   <signal>valueChanged(double)</signal>
   <receiver>deviceView</receiver>
   <slot>setPhi(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
